src/Maestro/Maestro.Data/Models/Build.cs

```diff
@@ -147,5 +150,7 @@ public class BuildDependency
         public int DependentBuildId { get; set; }
         public Build DependentBuild { get; set; }
         public bool IsProduct { get; set; }
+        [NotMapped]
+        public int Staleness { get; set; }
```

Comment:
This property doesn't look used.

Reply:
Thanks, I missed this when I cleaned up my changes.